README.md

```diff
@@ -371,6 +371,21 @@ This will change the prompt to:
 
 ![[master ≡] ~\GitHub\posh-git&#10;02-18 14:04:35 38> ][prompt-custom]
 
+Finally, the path portion of the prompt can be contained within delimiters.
+For instance, if you would like the containing characters to be red, curly braces, the following settings can be used:
+
+```text
+$GitPromptSettings.BeforePath = '{'
+$GitPromptSettings.AfterPath = '}'
+$GitPromptSettings.BeforePath.ForegroundColor = 'Red'
+$GitPromptSettings.AfterPath.ForegroundColor = 'Red'
+```
+
+With these additional values, the previous prompt would become
+
+![[master ≡] {~\GitHub\posh-git}&#10;02-18 14:04:35 38> ][prompt-custom-wpathdelim]
+
+
```

Comment:
Remove extra line here.

Reply:
Done

Reply:
I am a beginner when it comes to Github, so if I did not address your PR review comments, let me know.

Reply:
Thanks.  We appreciate the help on the README file.